src/site/content/en/blog/introducing-model-viewer/index.md

```diff
@@ -0,0 +1,174 @@
+
+### Camera controls
+
+`<model-viewer>` now gives full control over the view's virtual camera (the
+perspective of the viewer). This includes the camera target, orbit (position
+relative to the model), and field of view. You can also enable auto-rotation and
+set limits on user interaction (e.g. maximum and minimum fields of view).
+
+### Annotations
+
+You can also annotate your models using HTML and CSS. This capability is often
+used to "attach" labels to parts of the model in a way that moves with the model
+as it's manipulated. The annotations are customizable, including their
+appearance and the extent to which they're hidden by
+the model. Annotations also work in AR.
+
+```html
+<style>
+  button{
+    display: block;
+    width: 6px;
+    height: 6px;
+    border-radius: 3px;
+    border: 3px solid blue;
+    background-color: blue;
+    box-sizing: border-box;
+  }
+
+  #annotation{
+    background-color: #dddddd;
+    position: absolute;
+    transform: translate(10px, 10px);
+    border-radius: 10px;
+    padding: 10px;
+  }
+
+<model-viewer src="https://modelviewer.dev/assets/ShopifyModels/ToyTrain.glb" alt="A 3D model of a Toy Train" camera-controls>
+  <button slot="hotspot-hand" data-position="-0.023 0.0594 0.0714" data-normal="-0.3792 0.0004 0.9253">
+    <div id="annotation">Whistle</div>
+  </button>
+</model-viewer>
+```
+
+<!-- Hide until model-viewer's iframe bug is fixed -->
+<!-- <iframe style="width:100%; height: 100%;position: absolute; top: 50%; left: 50%; transform: translate(-50%,-50%);" src="https://modelviewer.dev/webdotdev/annotations.html" frameborder="0" allowfullscreen></iframe> -->
+
+<figure class="w-figure w-figure--inline-right">
+  <img src="./annotation.png" alt="A space suit with an annotation.">
+  <figcaption class="w-figcaption">A space suit with an annotation.</figcaption>
+</figure>
+
+See the [annotations documentation
+page](https://modelviewer.dev/examples/annotations.html) for more information.
+
+### Editor
+
+Version 1.1 introduces and hosts a `<model-viewer>` ["editing"
+tool](https://modelviewer.dev/editor/), which enables you to
+quickly preview your model, try out different `<model-viewer>` configurations
+(e.g. exposure and shadow softness), generate a poster image, and interactively
+get coordinates for annotations.
+
+### Rendering and performance improvements
+
+Rendering fidelity is greatly improved, especially for high dynamic range (HDR)
+environments. `<model-viewer>` now also uses a direct render path when only one
+`<model-viewer>` element is in the viewport, which increases performance
+(especially on Firefox). Lastly, dynamically scaling the render resolution
+improved frame rate dramatically. The example below shows off some of these
+recent improvements.
+
+```html
+<model-viewer camera-controls skybox-image="spruit_sunrise_1k_HDR.hdr" alt="A 3D model of a well-worn  helmet" src="DamagedHelmet.glb"></model-viewer>
+```
```

Comment:
```suggestion
```html
<model-viewer camera-controls 
              skybox-image="spruit_sunrise_1k_HDR.hdr"
              alt="A 3D model of a well-worn  helmet"
              src="DamagedHelmet.glb"></model-viewer>
```
```